custom_components/skydance/light.py

```diff
@@ -298,11 +298,31 @@ async def _set_rgbw(self, red: int, green: int, blue: int, white: int):
         _ = await self._session.read(64)
         self._attr_rgbw_color = red, green, blue, white
 
-    def _convert_color_temp(self, mireds):
-        """Convert color temperature from mireds to byte."""
+    @property
+    def min_color_temp_kelvin(self) -> int:
```

Comment:
Rather use this approach from docs:


from homeassistant.components.light import DEFAULT_MAX_KELVIN, DEFAULT_MIN_KELVIN

class MyLight(LightEntity):
    """Representation of a light."""

    # Old did not need to have _attr_min_mireds / _attr_max_mireds set
    # New needs to set the default explicitly
    _attr_min_color_temp_kelvin = DEFAULT_MIN_KELVIN
    _attr_max_color_temp_kelvin = DEFAULT_MAX_KELVIN